app/models/insured/factories/self_service_factory.rb

```diff
@@ -238,12 +248,20 @@ def self.eligible_for_1_1_effective_date?(system_date, current_effective_on)
 
       private
 
+      def fetch_aptc_value(enrollment, effective_on)
+        ::Operations::PremiumCredits::FindAptc.new.call({hbx_enrollment: enrollment, effective_on: effective_on}).value!
+      end
+
```

Comment:
clean way is to do memoization like below - and use this method whenever needed. this way if anyone wants to reuse this method, they don't have to make another call to this operation.

```
def fetch_aptc_value(enrollment, effective_on)
   @fetch_aptc_value ||= ::Operations::PremiumCredits::FindAptc.new.call({hbx_enrollment: enrollment, effective_on: effective_on}).value!
end
```